addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -476,6 +465,10 @@ import wUtils from '@website/js/utils';
         resetForm() {
             this.el.reset();
 
+            // Remove previous error Message as well.
```

Comment:
1. grammar fix :sweat_smile: 
`// Remove previous error messages.`

2. Here, for which case we will required to remove the errors manually? as `resetForm` aims to reset the input values.

Reply:
When error messages are present in dom and and you open the editor, those errors will be still present in viewport.